.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### Checklist
```

Comment:
Would you please make another PR for this targeting `main`? Since it does not apply when it is in `develop` branch.

Reply:
I'll take care of this.

Reply:
OK